dlrover/python/elastic_agent/torch/training.py

```diff
@@ -485,6 +486,8 @@ def launch_agent(
         f"  metrics_cfg      : {config.metrics_cfg}\n"
     )
 
+    ResourceMonitor()
+    logger.info("Elastic agent is imported ResourceMonitor")
```

Comment:
The log is not necessary. We can log in the `ResourceMonitor` to show we start monitoring workload.

Reply:
Fixed with 'Removed redundant logging, as well as improved monitor's startup log output.'